plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/ScanQueryPageSource.java

```diff
@@ -316,7 +316,7 @@ public void close()
                 }
                 catch (Exception e) {
                     // ignore
-                    LOG.debug("Error clearing scroll", e);
+                    LOG.debug(e, "Error clearing scroll");
```

Comment:
Can we handle the logger based changes in a separate commit ? The one where exception should be first and followed by message

Reply:
I split it into four commits. Thanks!